infer/src/IR/Procdesc.re

```diff
@@ -519,3 +519,35 @@ let node_set_succs_exn pdesc (node: Node.t) succs exn =>
     set_succs_exn_base node' [exit_node] exn
   | _ => set_succs_exn_base node succs exn
   };
+
+let get_loop_heads pdesc => {
```

Comment:
Ok, thanks. That makes sense. I should have paid more attention to what the code is doing.

Thanks for contributing this! Widening only at loop heads is a huge improvement over what we do today (widening every time a node is visited >1 time).